opm/parser/eclipse/EclipseState/EclipseState.hpp

```diff
@@ -123,7 +125,7 @@ namespace Opm {
 
 
         const UnitSystem& m_deckUnitSystem;
-        const ParseContext& m_parseContext;
+        const ParseContext m_parseContext;
```

Comment:
> Regarding your second point, the constructor now knows that its argument is a copy and will go out of scope with itself, so it should realise there's a move opportunity here, ensuring the actual copy is only performed once.

If I didn't understand this completely incorrectly, moves work by "capturing" the internal pointers of the source object in C++ (if the RHS is a rvalue reference). I.e., they're not "magical" and in particular they won't work if the object to be moved does not feature any internal pointers...